waitress/runner.py

```diff
@@ -171,6 +171,18 @@ def show_help(stream, name, error=None): # pragma: no cover
         print('Error: {0}\n'.format(error), file=stream)
     print(HELP.format(name), file=stream)
 
+
+def show_exception(stream): # pragma: no cover
```

Comment:
This pragma is not necessary -- it should be possible to write a unit test which exercises this function.